zeeguu/api/endpoints/translation.py

```diff
@@ -89,6 +91,7 @@ def get_one_translation(from_lang_code, to_lang_code):
             to_lang_code,
             context,
             article_id,
+            learning_cycle=LearningCycle.RECEPTIVE if _merle_exercises(flask.g.user) else LearningCycle.NOT_SET,
```

Comment:
Ok, so this should be now initialized to NOT_SET after the discussions of yesterday. right?

Reply:
Or just allow the default parameter value from the Bookmark constructor to take care of this. What I mean is that if you don't pass this, it's going to be set to NOT_SET anyway.